### DISCUSSION TITLE
Render content into a modal

### DISCUSSION THREAD

**Discussion by jackkitley** [inertiajs/inertia#1147] [Ideas]
We are using Inertia with Laravel 9. 
We have a few modals that need to be used with content from laravel routes.

When will there be support for modals? 

Currently im having to do custom get requests to get data but then it complains because its not a valid inertia response from laravel as im just outputting JSON.

Thanks

**Comment by ctessier** [comment]
See https://github.com/inertiajs/inertia/pull/642

This feature is in progress and yet to come 🤞
